### PR TITLE
#137085767 Export workouts to JSON

### PR DESCRIPTION
**Pull request by bmwenda**
#### What does this PR do?
Adds functionality to export workouts in a JSON file

#### Description of Task to be completed?
- Add a button that the user clicks to download workouts
- Enable the user to download their workouts in their local system

#### How should this be manually tested?
- Run python manage.py test

#### What are the relevant pivotal tracker stories?
#137085767

### CODE REVIEW

File: wger/manager/views/workout.py [hunk 1]
```diff
@@ -20,12 +20,16 @@
 
 from django.shortcuts import render, get_object_or_404
 from django.http import HttpResponseRedirect, HttpResponseForbidden
+from django.http import HttpResponse
```
Comment (osule) [inline wger/manager/views/workout.py]: We can combine imported classes from `django.http` here.

File: wger/manager/views/workout.py [hunk 2]
```diff
@@ -48,6 +52,7 @@
     WgerDeleteMixin
 )
 from wger.utils.helpers import make_token
+# from django.manager.serializers import WorkoutExportSerializer
```
Comment (osule) [inline wger/manager/views/workout.py]: Shall we be needing this commented out import?

File: wger/manager/views/workout.py (hunk 1 above)
Comment (bmwenda) [reply wger/manager/views/workout.py]: Okay, noted

File: wger/manager/views/workout.py [hunk 3]
```diff
@@ -20,12 +20,16 @@
...
+from django.http import HttpResponse
 from django.template.context_processors import csrf
 from django.core.urlresolvers import reverse, reverse_lazy
 from django.utils.translation import ugettext_lazy, ugettext as _
 from django.contrib.auth.mixins import PermissionRequiredMixin, LoginRequiredMixin
 from django.contrib.auth.decorators import login_required
 from django.views.generic import DeleteView, UpdateView
+# We will be able to create a ModelResource class
+# to describe how we export resource
+from import_export import resources
```
Comment (osule) [inline wger/manager/views/workout.py:32]: This looks to be unused. What the intent?

**Review by osule — CHANGES_REQUESTED** [review]

File: wger/manager/views/workout.py (hunk 3 above)
Comment (bmwenda) [reply wger/manager/views/workout.py:32]: It has been used the class that gets the items in the workout model as Json. Here: `class ExportWorkoutsResource(resources.ModelResource)`

File: wger/manager/views/workout.py (hunk 2 above)
Comment (bmwenda) [reply wger/manager/views/workout.py]: No, I'll get rid of that

**Review by osule — APPROVED** [review]
LGTM

**Pull request merged**